packages/commercetools/api-client/src/helpers/commercetoolsLink/index.ts

```diff
@@ -102,7 +102,26 @@ const createCommerceToolsConnection = (settings: Config): any => {
     delay: () => 0
   });
 
-  const apolloLink = ApolloLink.from([onErrorLink, errorRetry, authLinkBefore, authLinkAfter.concat(httpLink)]);
+  let links: ApolloLink[] = [
+    onErrorLink,
+    errorRetry,
+    authLinkBefore,
+    authLinkAfter.concat(httpLink)
+  ];
+
+  if (settings.overrideApolloLinks) {
+    links = settings.overrideApolloLinks({
+      sdkAuth,
+      tokenProvider,
```

Comment:
you are naming this `overrideApolloLinks` but you pass sdkAuth or tokenProvider

Reply:
maybe it's better to pass your own apollo client but with using links from this package?